openstates/ms/people.py

```diff
@@ -84,13 +84,27 @@ def scrape_details(self, chamber, leg_name, leg_link, role):
             home_address2 = root.xpath('string(//H_ADDRESS2)')
             home_city = root.xpath('string(//H_CITY)')
             home_zip = root.xpath('string(//H_ZIP)')
-
-            home_address_total = "%s\n%s\n%s\n%s" % (
-                home_address,
-                home_address2,
-                home_city,
-                home_zip
-            )
+            home_address_total = 'Not Available'
```

Comment:
instead of setting to Not Available, just leave it None (i.e. skip the add_contact_detail call)

Reply:
Ok